qtconsole/ansi_code_processor.py

```diff
@@ -292,6 +292,31 @@ def _replace_special(self, match):
             self.actions.append(ScrollAction('scroll', 'down', 'page', 1))
         return ''
 
+    def _parse_ansi_color(self, color, intensity):
+        """
+        Map an ANSI color code to color name or a RGB tuple.
+        Based on: https://gist.github.com/MightyPork/1d9bd3a3fd4eb1a661011560f6921b5b
+        """
+        parsed_color = None
+        if color < 16:
+            # Adjust for intensity, if possible.
+            if intensity > 0:
```

Comment:
@jsbautista I think we need to also add here a validation to not add `8` to an already brigth code color (`color` >=8; <16). So:
```suggestion
            if intensity > 0 and color < 8:
```